src/main/java/io/stargate/sgv2/dynamoapi/dynamo/TableProxy.java

```diff
@@ -22,10 +17,15 @@
 import java.util.Collection;
 import java.util.Date;
 import java.util.List;
-import javax.enterprise.context.ApplicationScoped;
+import java.util.stream.Collectors;
 
-@ApplicationScoped
 public class TableProxy extends Proxy {
+
+  private static final String SYS_RETURN_COLUMN = "table_name";
+  private static final String SYS_KEYSPACE = "system_schema";
+  private static final String SYS_TABLE = "tables";
+  private static final String SYS_COND_COLUMN = "keyspace_name";
```

Comment:
`SYS_RETURN_COLUMN` and `SYS_COND_COLUMN` are confusing. I would simply use `TABLE_NAME` and `KEYSPACE_NAME` instead.

Reply:
OK.